conference/src/test/java/org/spine/samples/lobby/conference/ConferenceServiceShould.java

```diff
@@ -212,5 +219,141 @@ public Void apply(Void input) {
                                         .build(), conferenceUpdatedObserver);
     }
 
+    @SuppressWarnings("unchecked")
+    @Test
+    public void publish_conference_and_generate_ConferencePublished_event() {
```

Comment:
The ideal test method looks like this (and actually is really short):

```
// given
TestObj obj = givenTestObj();

// when
obj.doSmth();

 // then
assertEquals(expected, obj.getActual());
```

Three lines is cool, in some cases there can be even one or two.
`The art of unit testing` is a great book to read on this topic.